recipes-wpe/wpeframework/wpeframework_git.bb

```diff
@@ -1,8 +1,20 @@
+HOMEPAGE = "https://github.com/rdkcentral/Thunder"
 SUMMARY = "Web Platform for Embedded Framework"
 DESCRIPTION = "A C++ platform abstraction layer for generic functionality."
 
+SECTION = "wpe"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=85bcfede74b96d9a58c6ea5d4b607e58"
+
+RECIPE_BRANCH ?= "development/cmake-tools-path-settings"
```

Comment:
Please do not bump to changes present only on the development branch.
If you need to use dev branch please use .bbappend on you platform layer.

Reply:
Sorry, I missed to update this. will correct

Reply:
Please correct that.

Reply:
updated